libs/sdk-ui-tests/stories/_infra/webComponents.ts

```diff
@@ -0,0 +1,32 @@
+// (C) 2022 GoodData Corporation
+/**
+ * This file does not export and only configures the mock backend with Web Components,
+ * as a side effect. Backend only has to be defined once per runtime.
+ */
+
+import { IAnalyticalBackend } from "@gooddata/sdk-backend-spi";
+import { StorybookBackend, ReferenceWorkspaceId } from "./backend";
+
+const configureWebComponentsBackend = (backend: IAnalyticalBackend, workspaceId: string) => {
+    // preview-head.html loads after this file.
+    // I.e. by the time this runs the global variables is not set yet
+    const configureBackend = () => {
+        if (window.__setWebComponentsContext) {
+            try {
+                window.__setWebComponentsContext({
+                    backend: backend,
+                    workspaceId: workspaceId,
+                });
+            } catch (e) {
+                // setContext will throw if the context was already set...
+                clearInterval(checkInt);
+            }
+            clearInterval(checkInt);
+        }
+    };
+
+    const checkInt = setInterval(configureBackend, 100);
+};
+
+// Configure the mock backend
+configureWebComponentsBackend(StorybookBackend(), ReferenceWorkspaceId);
```

Comment:
This is an (ugly) fix that we discussed with Dan and Matyas. Needed due to the WebPack limitations on loading of the chunked libs.
The file does not export anything, sets up the mock backend for WebComponents as a side effect. This is done because the backend setup is only done once per runtime by design.